src/v2/styles/ImageCarousel/ImageCarousel-theme.scss

```diff
@@ -1,10 +1,16 @@
 @use '../utils';
 
 .str-chat {
+  --str-chat__image-carousel-track-background-color: var(--str-chat__background-color);
```

Comment:
@MartinCupela can you please add CSS-style comments `/* ... */` with the descriptions of the new variables? Component doc generation will be merged soon.